Modules/rat_command.py

```diff
@@ -129,6 +136,9 @@ def _register(cls, func, names: list or str) -> bool:
         if isinstance(names, str):
             names = [names]  # idiot proofing
 
+        # transform commands to lowercase
+        names = [x.lower() for x in names]
```

Comment:
Usage of a single char variable. Given the context, im going to have to ask you to change this.

Reply:
Also updated, new variable is lowered_name.